lib/plaid/products/asset_report.rb

```diff
@@ -63,12 +63,17 @@ def refresh(asset_report_token, days_requested, options = {})
     # Public: Retrieve an asset report.
     #
     # asset_report_token - The asset report token from the `create` response.
+    # include_insights   - An optional boolean specifying whether we should
+    #                      retrieve the report as an "Assets + Insights"
+    #                      report. For information about Assets + Insights
+    #                      reports, see https://plaid.com/docs/#assets.
     #
     # Returns a AssetReportGetResponse object.
-    def get(asset_report_token)
+    def get(asset_report_token, include_insights = false)
```

Comment:
Let's use a named argument here: `def get(asset_report_token, include_insights: false)`. We do this [elsewhere](https://github.com/plaid/plaid-ruby/blob/5a2c1c93aa6bf66a7cd7519984b871c44ec26249/lib/plaid/products/item.rb#L216-L223).